BUILDING.md

```diff
@@ -15,6 +15,15 @@ or MonoDevelop.
 
 The solutions all place their output in a common bin directory under the solution root.
 
+## Running Tests
+
+The tests that should be run in the solution are grouped by project name:
+ * `nunit.framework.tests-*`
+ * `nunitlite.tests-*`
+ * `slow-nunit-tests-*`
+
+Other test projects contain tests designed to fail purposely for integration tests.
```

Comment:
Not precisely true, unless you consider some of our unit tests to be integration tests. Up to now, we have merely considered them as bad unit tests. 😄 We need separate integration tests for sure.

IMO, this is the problem you will run into when documenting a moving target. 😢